packages/transport-bridge/src/core.ts

```diff
@@ -79,7 +79,27 @@ export const createApi = (apiStr: 'usb' | 'udp') => {
             return enumerateResult;
         }
 
-        return sessionsClient.enumerateDone({ paths: enumerateResult.payload });
+        // todo: one mapping here and one mapping below, maybe this could be improved
```

Comment:
the other option is to sent full descriptor with everything to sessions background. 

PROS:
- less mapping back and forth like here

CONS: 
- sessions background doesn't need to know anything about this kind of information

also what I dont like in general about the field `descriptor.type` is that I see `descriptor` as "the minimal information needed  to identify and talk to device`. This is already some kind of meta information - it maybe fits but I am not sure if fully.

Reply:
I am slightly in favor of changing it in that way that background would receive full descriptors instead sessions only but I would like to do it in a subsequent PR